draft-packages/loading-spinner/KaizenDraft/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -10,17 +10,15 @@ export interface LoadingSpinnerProps {
    * @default "md"
    */
   size?: size
-  children?: React.ReactNode
```

Comment:
I think I remember @HeartSquared mentioning this at some point, and was wondering if we want to trigger a breaking change here.

I've just had a look through all of the usages in our FE repos and no one is passing children, so we're safe to change this without a breaking change 👍

Reply:
I believe I was talking about Radio or something, but yes it might be happening to some others 😂 If you feel it's safe, remove away! :)